pointerevents/pointerevent_capture_suppressing_mouse-manual.html

```diff
@@ -89,15 +90,15 @@
                         test_pointerover_capture.done();
                         if (!isRelatedTargetValueTested) {
                             test(function() {
-                                assert_true(event.relatedTarget==null, "relatedTarget is null when the capture is set")
-                            }, "relatedTarget is null when the capture is set. relatedTarget is " + event.relatedTarget);
+                                assert_true(event.relatedTarget != null, "relatedTarget should behave the same as when the capture is not set")
```

Comment:
nit: you get better errors if you [use assert_not_equals](https://github.com/w3c/testharness.js/blob/master/docs/api.md#list-of-assertions)